sdcm/cluster.py

```diff
@@ -1595,6 +1595,32 @@ def config_setup(self,
                 self.parent_cluster.proposed_scylla_yaml,
                 self.proposed_scylla_yaml
             )
+            is_kms = bool(scylla_yml.kms_hosts)
+
+        if is_kms:
+            # Hack for overriding issue https://github.com/scylladb/scylla-enterprise/issues/2792
+            # TODO: should be remove once a proper fix is implemented
+            self.remoter.sudo("find /opt/scylladb/ -iname *libp11-kit.so* | sudo xargs rm",
```

Comment:
Shouldn't it consider distro? As I understood, it must be run only on Ubuntu/Debian, right?